firmware/spade/src/rpi/main.c

```diff
@@ -7,6 +7,10 @@
 #include "pico/util/queue.h"
 #include "pico/multicore.h"
 #include "jerryscript.h"
+#include "shared/FatFs_SPI/include/f_util.h"
```

Comment:
might be good to put this within its own standlone .h just for ease of organization of things

Reply:
i.e. all of these fatfs things